src/common/transformations/include/transformations/hash.hpp

```diff
@@ -34,5 +34,26 @@ class NGRAPH_API Hash : public ov::pass::ModelPass {
     uint64_t& m_hash;
 };
 
+/**
+ * @brief Hash transformation for faster calculation of hash value for ov::Model
+ */
+class NGRAPH_API FasterHash : public ov::pass::ModelPass {
```

Comment:
Can we move this class to separate header?

Reply:
I don't think that we have to check smth because all public transformations are located in `src/core/include/openvino/pass` folder and all other transformations are located in common and they are not public, so they don't have this macros.

Reply:
I'm not sure about new header - I believe at some point we'll remove 'HashPass' and rename 'FasterHash' to 'HashPass'. It would be good to avoid renaming back `#include <faster_hash.hpp>` to `#include <hash.hpp>` in client's code (tests)
Regarding 'export' - I believe it can be done under some separate cleanup activity. Existing 'ieFuncTests' use such transformations API directly, so exporting this is needed for tests. So it is needed to rework current tests to avoid NGRAPH_API exports (or maybe move them from ieFuncTests to ieUnitTests)